src/sass/ResultActionMenu.scss

```diff
@@ -51,7 +53,6 @@ $tooltip-z-index: 1000;
         height: $menu-height;
         position: relative;
         box-sizing: border-box;
-        border: thin solid $heather;
```

Comment:
Why no border?

Reply:
If you look closely at the before Screenshot, there are 2 different borders overlapping